.github/workflows/ci.yml

```diff
@@ -28,7 +28,8 @@ jobs:
           java-version: ${{ matrix.jdk }}
 
       - name: Check formatting
```

Comment:
The name doesn't look accurate anymore. Maybe "Check formatting and run linter"?

Reply:
Yes. That seems like a better name.